keyboards/blaster75/keymaps/via/keymap.c

```diff
@@ -0,0 +1,51 @@
+/* Copyright 2021 by Altain
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_all(
```

Comment:
`LAYOUT_all` does not exist

Reply:
Thanks for the guidance. I'll make the corrections swiftly. So it should just be "LAYOUT(" for via keymap?

Reply:
It needs to compile. Which requires it to be a valid layout macro defined the keyboards `<keyboard>.h` of which `LAYOUT_all` is not one of `LAYOUT`, `LAYOUT_blaster_75_ansi` or `LAYOUT_blaster_75_7u`.

Reply:
Ahh, so since it's via keymap it had no problem on my side when I was just compiling with QMK then. I'll make the adjustment asap.

Reply:
Oh, turns out by just deleting "_all" parts I did exactly that on the last commit. Thank you so much for explanation! Now it makes sense when I look back at it.

Reply:
@zvecr So I've also adjusted via/keymap.c files to your advice, and I've gone through my own lint check and it passed. What's the next step I should take on? Really sorry to bother you.